packages/ecr-credential-provider-1.31/ecr-credential-provider-1.31.spec

```diff
@@ -2,7 +2,7 @@
 %global gorepo cloud-provider-aws
 %global goimport %{goproject}/%{gorepo}
 
-%global gover 1.31.4
+%global gover 1.31.5
```

Comment:
Please fix `GO_VERSION` in this package to use `GO_MAJOR` instead. Might as well also fix the k8s 1.31 spec which is the only other user besides this one.